service/samples/registry_bulk_identity_sample.js

```diff
@@ -0,0 +1,131 @@
+                       console.error('Could not create output container: ' + error.message);
+                   } else {
+                       var outputSasToken = blobSvc.generateSharedAccessSignature(outputContainerName, null, outputSharedAccessPolicy);
+                       var outputSasUrl = blobSvc.getUrl(outputContainerName, null, outputSasToken);
+
+                    /**
+                     * There can only be one active job at a time, therefore, you can uncomment the export section and comment the import
+                     * session or vice versa depending on what part of the code you would like to test.
+                     */
+
+                    //    registry.exportDevicesToBlobByIdentity(outputSasUrl, false, function (error, result) {
+                    //        if (error) {
+                    //            console.error('Could not create export job: ' + error.message);
+                    //        } else {
+                    //            console.log('--------------\r\nDevices Export Job Identifier:--------------\r\n' + result);
+                    //            var jobId = result.jobId;
+                    //            var interval = setInterval(function () {
+                    //                registry.getJob(jobId, function (error, result) {
+                    //                   if (error) {
+                    //                       console.error('Could not get job status: ' + error.message + ' : ' + error.responseBody);
+                    //                   } else {
+                    //                       console.log('--------------\r\njob ' + jobId + ' status:\r\n--------------\r\n' + result);
+                    //                       var status = result.status;
+                    //                       if (status === "completed") {
+                    //                           clearInterval(interval);
+                    //                       }
+                    //                   }
+                    //               });
+                    //            }, 500);
+                    //        }
+                    //    });
+
+
+                       registry.importDevicesFromBlobByIdentity(inputSasUrl, outputSasUrl, function (error, result) {
+                          if (error) {
+                              console.error('Could not create import devices: ' + error.message + ' : ' + error.responseBody);
+                          } else {
+                              console.log('--------------\r\nDevices Import Job Identifier:--------------\r\n' + result);
+                              var jobId = result.jobId;
+                              var interval = setInterval(function () {
+                                  /**
+                                   * Uncomment this code to test cancelling a job.
+                                   */
+                                //   registry.cancelJob(jobId, function (error, result) {
```

Comment:
No commented code - again, you can use args.  We tried to keep our samples such that they never have to be edited.